pliant/README.md

```diff
@@ -27,32 +27,34 @@ Create one or more workflows that you would like to trigger from a Datadog notif
 
 ![API Key Menu step1][8]
 
-2. From the API keys screen, click "+ Create" at the top right of the screen and title your new API key. Click save and note the the API key will be added to the table.
+2. From the API keys screen, click "+ Create" at the top right of the screen and title your new API key. Click save and make note of the API key, which will be added to the table.
 
 ![Create API Key step2][9]
 
 **Create a Pliant workflow to trigger from Datadog**
 
-1. Navigate to the workflows tab in Pliant. Click "+ Create"  and "Create Flow" to create a new workflow. Title the workflow in the popup and click "Create" to launch the editor into our new workflow.
+1. Navigate to the workflows tab in Pliant. Click "+ Create" and "Create Flow" to create a new workflow. Title the workflow in the popup and click "Create" to launch the editor into the new workflow.
 
 ![Create Flow step1-a-][10]
 
 2. Populate the workflow with actions to take upon receiving the Datadog trigger. 
 
 This example workflow is called "RestartHost" and restarts a host from the data Datadog triggers this workflow with.
 
-This workflow will run with its input variables initially assigned based on the request body we trigger it with. The workflow can trigger/perform any desired infrastructure automation actions, using information from its input. In this example, restart a host via SSH under certain circumstances when Datadog triggers our automation workflow with certain parameters. 
+This workflow runs with its input variables initially assigned based on the request body you trigger it with. The workflow can trigger/perform any desired infrastructure automation actions, using information from its input. In this example, restart a host with SSH under certain circumstances when Datadog triggers the automation workflow with certain parameters.
 
   - To add Input variables which populate with data sent from Datadog, click the "Expand" icon on at the start of the workflow to open the Variable panel. To create matching **Input** variables, set all of these input variables to equal empty quotes: `""`. By default, Datadog sends the following data:
-`body`
-`last_updated`
-`event_type`
-`title`
-`date`
-`org`
-`id`
-
-There are also have additional output variables (`host`, `meta`, and `ip`) that are initialized. The workflow will assign these output variables and output the resulting values upon completion. It may also specify variables which are neither input nor output variables to use internally within the workflow's logic.
+    ```
+    body
+    last_updated
+    event_type
+    title
+    date
+    org
+    id
+    ```
+
+There are also have additional output variables (`host`, `meta`, and `ip`) that are initialized. The workflow assigns these output variables and outputs the resulting values upon completion. It may also specify variables which are neither input nor output variables to use internally within the workflow's logic.
```

Comment:
```suggestion
There are also additional output variables (`host`, `meta`, and `ip`) that are initialized. The workflow assigns these output variables and outputs the resulting values upon completion. It may also specify variables which are neither input nor output variables to use internally within the workflow's logic.
```